lib/scripts.js

```diff
@@ -1,5 +1,29 @@
 'use strict';
 
+const parseDataAttributes = (scriptObj) => {
+
+    // Don't mutate the original object.
+    const script = Object.assign({}, scriptObj);
+
+    if (!script.dataAttributes) {
```

Comment:
I think this should be above 6. You're actually, needlessly, duplicating and returning a different object by doing it this way.

Reply:
@smebberson With the Jade change below I think we need to keep this here and add a `script.dataAttributes` default.

Reply:
Yep, makes sense.